notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -28,10 +28,10 @@
     "- [Select model for inference](#Select-model-for-inference)\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">#&nbsp;TODO: remove compression via NNCF for non-MPT models when INT4 weight compression is added to optimum-intel
</pre><p>This should have been kept? (ticket 124681) But not critical. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1492/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>